java/src/jmri/jmrit/withrottle/TurnoutController.java

```diff
@@ -53,41 +54,60 @@ public void filterList() {
         sysNameList = tempList;
     }
 
+    /**
+     * parse and process a turnout command message
+     * <p>
+     * Format: PTA[command][turnoutname]
+     *   where command is 'C'losed, 'T'hrown, '2'oggle and
+     *   turnoutname is a complete system name or a turnout number only
+     *     if number only, system prefix and letter will be added
+     * Checks for existing turnout and verifies it is allowed, or  
+     *   if Create Turnout preference enabled, will attempt to create turnout
+     * Then sends command to alter state of turnout
+     * Can return HM error messages to client
+     * @param message Command string to be parsed
+     */
     @Override
     void handleMessage(String message) {
         if (message.charAt(0) == 'A') {
-            //first look for existing turnout
-            Turnout t = manager.getBySystemName(message.substring(2));
+            String tName = message.substring(2);
```

Comment:
It's not necessarily the case that the system name starts with a two-character prefix; they can be longer than that.

Not entirely sure what you're trying to do here, but the general idea of `provideTurnout` is that it handles numerical inputs going to the default system, etc.

Reply:
@bobjacobsen 
That removal of the 2 characters is not doing anything with the system prefix, that's part of the command passed in from the client. 'AT', 'AC' or 'A2'. The system prefix (or lack thereof), is included in tName. I've tested with 1, 2 and 3 character system prefixes, and with no prefix at all.